mllib/src/main/scala/org/apache/spark/ml/feature/LSH.scala

```diff
@@ -147,6 +178,18 @@ private[ml] abstract class LSHModel[T <: LSHModel[T]]
     modelSubsetWithDistCol.sort(distCol).limit(numNearestNeighbors)
   }
 
+  /**
+   * Overloaded method for approxNearestNeighbors. Use 0.05 as default relativeError.
+   */
+  private[feature] def approxNearestNeighbors(
```

Comment:
Rather than make a new method, just add a new optional arg for error. The change looks OK. But this isn't actually exposed then?